### PR TITLE
[quest] Cata Northern Stranglethorn prequest fixes

### LINKED ISSUE #6276: Quest Issue - Bloodscalp Insight (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [The Defence of Grom'gol: Murloc Menace](https://www.wowhead.com/cata/quest=26280/the-defense-of-gromgol-murloc-menace) and [The Defence of Grom'gol: Ogre Oppression](https://www.wowhead.com/cata/quest=26279/the-defense-of-gromgol-ogre-oppression)

## Wowhead link

https://www.wowhead.com/cata/quest=9436/bloodscalp-insight

### LINKED ISSUE #6277: Quest Issue - Hunt for Yenniku (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [The Defence of Grom'gol: Murloc Menace](https://www.wowhead.com/cata/quest=26280/the-defense-of-gromgol-murloc-menace) 
## Wowhead link

https://www.wowhead.com/cata/quest=26298/hunt-for-yenniku

## Questie & WoW version

Cata Classic
Questie 10.3.7

### LINKED ISSUE #6278: Quest Issue - A Nose For This Sort of Thing (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [Favored Skull (Horde)](https://www.wowhead.com/cata/quest=26323/favored-skull) or [Favored Skull (Alliance)](https://www.wowhead.com/cata/quest=26745/favored-skull)

## Wowhead link

https://www.wowhead.com/cata/quest=26325/a-nose-for-this-sort-of-thing

### LINKED ISSUE #6280: Quest Issue - Nezzliok Will Know (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [Headhunting](https://www.wowhead.com/cata/quest=26299/headhunting) 

## Wowhead link

https://www.wowhead.com/cata/quest=26300/nezzliok-will-know

## Questie & WoW version

Cata Classic
Questie 10.3.7

### LINKED ISSUE #6281: Quest Issue - Speaking with Nezzliok (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [Nezzliok Will Know](https://www.wowhead.com/cata/quest=26300/nezzliok-will-know)

## Wowhead link

https://www.wowhead.com/cata/quest=26301/speaking-with-nezzliok

## Questie & WoW version

Cata Classic
Questie 10.3.7

### LINKED ISSUE #6287: Quest Issue - Bloodlord Mandokir (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [Speaking With Nezzliok](https://www.wowhead.com/cata/quest=26301/speaking-with-nezzliok) and [Skullsplitter Mojo](https://www.wowhead.com/cata/item=58201/skullsplitter-mojo)

## Wowhead link

https://www.wowhead.com/cata/quest=26334/bloodlord-mandokir
## Questie & WoW version

Cata Classic
Questie 10.3.7

### LINKED ISSUE #6294: Quest Issue - Zul'Mamwe Mambo (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [See Raptor](https://www.wowhead.com/cata/quest=26359/see-raptor)
## Wowhead link

https://www.wowhead.com/cata/quest=26405/zulmamwe-mambo

## Questie & WoW version

Cata Classic

### LINKED ISSUE #6295: Quest Issue - Nighttime in the Jungle (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [See Raptor](https://www.wowhead.com/cata/quest=26359/see-raptor)

## Wowhead link

https://www.wowhead.com/cata/quest=26304/nighttime-in-the-jungle

## Questie & WoW version

Cata Classic
Questie 10.3.7

### LINKED ISSUE #6300: Quest Issue - Diffractory Chromascope (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [The Holy Water of Clarity](https://www.wowhead.com/cata/quest=26433/the-holy-water-of-clarity)

## Wowhead link

https://www.wowhead.com/cata/quest=26592/diffractory-chromascope

## Questie & WoW version

Cata Classic
Questie 10.3.7

### LINKED ISSUE #6301: Quest Issue - Primal Reagents of Power (Cata Classic)
## Description

Showing as available before pre-quests are completed. Requires [The Holy Water of Clarity](https://www.wowhead.com/cata/quest=26433/the-holy-water-of-clarity)

## Wowhead link

https://www.wowhead.com/cata/quest=26434/primal-reagents-of-power
## Questie & WoW version

Cata Classic
Questie 10.3.7

### PR DESCRIPTION
**Pull request by coldpasta123**
## Issue references

Fixes #6295 
Fixes #6301 
Fixes #6300 
Fixes #6294 
Fixes #6287 
Fixes #6281 
Fixes #6280 
Fixes #6278 
Fixes #6277 
Fixes #6276 

## Proposed changes
Fix prequests for:
- Quest Issue - Nighttime in the Jungle (Cata Classic) #6295
- Quest Issue - Primal Reagents of Power (Cata Classic) #6301
- Quest Issue - Diffractory Chromascope (Cata Classic) #6300
- Quest Issue - Zul'Mamwe Mambo (Cata Classic) #6294
- Quest Issue - Bloodlord Mandokir (Cata Classic) #6287
- Quest Issue - Speaking with Nezzliok (Cata Classic) #6281
- Quest Issue - Nezzliok Will Know (Cata Classic) #6280
- Quest Issue - A Nose For This Sort of Thing (Cata Classic) #6278
- Quest Issue - Hunt for Yenniku (Cata Classic) #6277
- Quest Issue - Bloodscalp Insight (Cata Classic) #6276

### CODE REVIEW

File: Database/Corrections/cataQuestFixes.lua [hunk 1]
```diff
@@ -4100,6 +4119,10 @@ function CataQuestFixes.Load()
         [26333] = { -- No Tanks!
             [questKeys.zoneOrSort] = zoneIDs.CHILL_BREEZE_VALLEY,
         },
+        [26334] = { -- Bloodlord Mandokir
+            [questKeys.preQuestSingle] = {},
+            [questKeys.preQuestGroup] = {26301,26332},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:4124]: Where did you find that information?

File: Database/Corrections/cataQuestFixes.lua [hunk 2]
```diff
@@ -546,6 +546,10 @@ function CataQuestFixes.Load()
         [9369] = { -- Replenishing the Healing Crystals
             [questKeys.requiredRaces] = raceKeys.ALL_ALLIANCE - raceKeys.DRAENEI,
         },
+        [9436] = { -- Bloodscalp Insight
+            [questKeys.preQuestSingle] = {},
+            [questKeys.preQuestGroup] = {26280,26279},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:551]: Same here. How do you know both quests are required for the pick up?

Comment (Cabro) [reply Database/Corrections/cataQuestFixes.lua:551]: Easiest way to check for this is to turn in the quest that is labeled prequestsingle in quest `first`. If the followup doesn't pop, then it's a prequestgroup

Comment (coldpasta123) [reply Database/Corrections/cataQuestFixes.lua:551]: I tested this with two characters, turning in a different quest first each time - the follow up didn't appear until both were handed in. I will try a third character tomorrow just to be sure.

**Comment by USCHHR** [comment]
Lack of tasks 83714

File: Database/Corrections/cataQuestFixes.lua (hunk 2 above)
Comment (coldpasta123) [reply Database/Corrections/cataQuestFixes.lua:551]: Retested with another two characters - the prequests are actually The Defense of Grom'gol: Murloc Menace (26280) and I Think She's Hungry (26321)

File: Database/Corrections/cataQuestFixes.lua (hunk 1 above)
Comment (coldpasta123) [reply Database/Corrections/cataQuestFixes.lua:4124]: Retested - just https://www.wowhead.com/cata/quest=26332/skullsplitter-mojo required

**Pull request merged**